packages/sit-onyx/src/components/examples/GridPlayground/GridPlayground.vue

```diff
@@ -264,6 +264,8 @@ const currentBreakpoint = computed(() => {
   font-family: var(--onyx-font-family);
   color: var(--onyx-color-text-icons-neutral-intense);
   background-color: var(--onyx-color-base-background-tinted);
+  overflow: hidden;
```

Comment:
My bad, using `overflow: hidden` doesn't allow more than two rows.
But we can fix this by using `scroll` here and adding `overflow: hidden` to the `div`, which contains the `GridOverlay`:
![Screenshot 2025-03-06 at 10 03 54](https://github.com/user-attachments/assets/b4790357-5ab5-4220-875c-77b6c070d906)



```suggestion
  overflow: scroll;
```

Reply:
Instead of style I added a class because style attributes were not allowed.